ibis_ml/steps/_impute.py

```diff
@@ -71,11 +71,10 @@ def _stat(self, col: ir.Column) -> ir.Scalar:
 
     def fit_table(self, table: ir.Table, metadata: Metadata) -> None:
         columns = self.inputs.select_columns(table, metadata)
-        self.fill_values_ = (
+        self._fit_expr = [
             table.aggregate([self._stat(table[c]).name(c) for c in columns])
-            .execute()
-            .to_dict("records")[0]
-        )
+        ]
+        self.fill_values_ = self._fit_expr[-1].execute().to_dict("records")[0]
```

Comment:
```suggestion
        self.fill_values_ = self._fit_expr[0].execute().to_dict("records")[0]
```
If it's the only element, more normal to access as `[0]`